content/docs/1.9.0/v2-data-engine/quick-start.md

```diff
@@ -159,6 +159,11 @@ INFO[2024-01-01T00:00:10Z] Cleaning up preflight installer
 INFO[2024-01-01T00:00:10Z] Completed preflight installer. Use 'longhornctl check preflight' to check the result.
 ```
 
+> **Note**:
+> Some immutable Linux distributions, such as SUSE Linux Enterprise Micro (SLE Micro), require you to reboot worker nodes after running the `install` sub-command. After the reboot, you will have to run the `install` sub-command again to complete the operation.
```

Comment:
Do we know why need to execute `install` command again? I thought rebooting should be able to take effect after running `install` command? Or the second command is more about checking if the required dependencies are installed successfully?

Reply:
No need. The PR here should be good enough.